mappings/69df7562f6c86d30b821b9824ee5ef3284ae01f88f4a51deeb148cf14772616365.json

```diff
@@ -0,0 +1 @@
wsLyNQbO/tM5gM0dowiRI6+33kUnuBRtXHDVzubO6xs9+hyFOW5udZas+xs71NFCe4jkORl2sRfNfDkQblSnIz21BpkLI8Ylqqspw1218spmYSBME05zPluZFyFtnKow6LvChbJiXTjXHm8KzcNE2ZTCbT01XLDeVlgXYKUaa+MElytjb3cRxJs9kiiQ2D0Yi0KGe/HcdlNJ6Qp2UmIaXE9dzDaNvtHSCkZXl5kVajzurSAovz8wyHY7TJcLCGxYUFbt3eoF6r43mKdrOGkpL+YEijWaNas1g0ozghigoKk/PEI6e50Tng/c0x1rrl+Y4ClFO2NzqOwvdm52zPBpzEBzq0dW6mb7igMJo8Lsf8lHIPm62yLDukdZQqKeryNAVnmgrlU7pZoVQZeIKgQrUW0u32SYsUrGE8TDh57DijSUQapwRuyHgwpBJWiJOkTM0clzwz9A72qVTK5ZZ5lqEcaDbDsuciywouvX+V1eVllCpbm9wgpNPpMT/XQDqK3nDEJIrp9kd4vo8Vghu3dqj7oIwh1RLPDfAciVQlgCzyMcaWHYuluZXAdNZCVR4rDUWhyTNNoQ1xlFEJK1jK84aCaYCYLRYX01UweZFTqQSALavRSuIFIUpBUSR04hFiR5V7dKyh4nlUg3ItTC2swOLSNGOwFEVOkqTT4yymWzVNuQp+HCe4QpTpmOdQ9T3k9t4eVgj6g0FJ1GHpdLvTczEEUZyys9ulN4xoN5o8+ch93LN2nE4nwVEJH7p3DkdPpnffKQueU2yltaAoQBclFptBCddVOK7C8x0838VqkCYnzROMUfjeNGgUBbNDBjzPOzxE2FEKx3Gnx75aPM9HUfa+Br6L47p0u0O63dFUAII8y1GOKoUdBiglOLa6zPx8i2Nrx3Cckv6uVsrc1PPdcskRsL/fYTyaoE2BIx0XLwyohBX6vQnShYXWHKNJzPU72xSmTIvSLOPcieN0unvksebBCydxA4fRaEQtVLx2MyaXHhWnSpxkSMryW1FopCyzATvtcpwJwXUU2vNBjLlwtkle1HjtnXX8oF0eCGzKPdFSSvKiOGqbnB5qHkVJue7YdXFkOZ44V3ep+hKDII5TRKWOsTnd/oCGreJ7LoHvk6YZhcyxaGqNkFptGc/xGQ1G+IGHNiVZ0G410UVBkmTleeNJnDIZT0CUDOPScptasw7TSlJRlKDznlNruJ4gzgx7nS5SKfq9Ia1ajXtOL5EaW44RaqhU63iOX3Y1Yg4PlGLqr6QVOFLB9PT2MHQ4f2KBLzz/BI8/fJYsz1BKokQ5cpzn+WEUFVYgVdlRowtLpVIHK7BSkecZQlgWFtosryyTFhkCzeJii3qzhpCSvMgZDsfkuWFv74DRIGJna4ckiun3+xwc9IjjmEoYIoWhWg1wXEW1HhLWKsh6s4pUkjsb2/QnfTJj2NvdY3W1jeMrqpWA++69h1aziiNddrZ7rB5bodsfkBcanRdEkxRHOhS5KXeqeh65LY62WJrpEdRTWKGcsrRmRak5J1fqtBs1WnMVvvwzH6VZD1COQusMIU3pX6bpkzGmXBVaFHhBeVpItx9z0I/xPYf2YousSNjd3SWOM+qNKq12g3qjhnIkS8srzM01yYuMaJLS7Y7odoZMRglpkiEdRaVSYTIacvbkSbq9AfGU4fU8BznfmqPiBfheSJHBxu1deoMxkyQmjiYcW14Ao4nTlMFwSKvRZDgc4lkDVvHO1VsMR5Ny64hSuG6AmTp66Th4QTidny7KIuy0/VMqcXiUdLsmydKUzs4Aq3NOrC0wmYzLvNAc4TfHcXC9WYT00NowHo2wjo/jV/EUpHGM7wbkuSFLU/Ikpd8dooSiWW/R7XSIkwm1esBDj9zL0nKLer3OoD8hmiRYbRkMhkTRZAqkc+IkJ4sLdrb2kaNeH6ELzpw8SeiHhH6Acj2uXbvN4sICge9Qb9QJqhUQ0Kh5KOkS1hokcUq1XgFl0ImmQCGVW2Iv6aC8AOUG04N6NZ6jMHp6dIYoHXCrqlieb+IFIXvdIW++c5GPPXE/860WWZEz6+bO8xws035TH2vL02AKbQkqdTxl8ZQljhL29wd0B0OWF+dpNZtIIUjjCQJDJayAlWAlve4BiJyTJ1cIKx5+4E2DX1mJ6vYHTMYTTF5gtSnBtkBQa9To9rpkWUaj2SCOY4LAZzwacWdzm87BARQF9VqTfn/I8soCWzs7IDOWF+dRaI41S3ykhYNQCpSLdPySlnbk4XrrWZe21hppDfefXaRa9QnDAL8SEgYe40HJw9eqIdYWhwl6XuTEacYkikmzlDTT4FXwHIFOh1QqPsNJzPZBj15/gOe6KEfheS6u5zAYlFNqSZIgRFkvyNKcrc1NgtBlabnF0mKbwPdI0oQoipHSKYG1LV3B/x8nORJ4oc9UlgAAAABJRU5ErkJggg==", "signatures": [{"signature": "2959fb78fb375b7a94f9ca1a428e8f1323ac6cdd7154da1460ecf229b330b2589df01a817c2575b5461ec495617c5dd7da0bc41a3b1fbe86abd46482bed8860b", "publicKey": "ec731c2d528c9dba869363a164084024416ec8431d50b6d9359078c58a70a36e"}]}, "description": {"sequenceNumber": 0, "value": "Nike Grace and Charles\n", "signatures": [{"signature": "04bf5d0feb428b77a195f612b0186f8070624fedebf5290a39f04dbc98f017771cc02cbf601257b5fd06cea5fdd77dc640636c5a5fdb1af02f93422601f3560e", "publicKey": "ec731c2d528c9dba869363a164084024416ec8431d50b6d9359078c58a70a36e"}]}}
```

Comment:
Hi @CardanoTokenIO,

please change the logo. Charles Hoskinson can't be used as logo for a token

Thanks!